Inseok_45/controllers/userController.js

```diff
@@ -20,4 +25,18 @@ const signUp = async ( req, res ) => {
     };
 };
 
-module.exports = { signUp };
\ No newline at end of file
+const getUserById = async (req, res) => {
+    try{
+        const { userName, password } = req.body;
+        if(!userName || !password) return res.status(400).json({ message: "INVALID_KEY" });
+
+        const token = await userService.getUserById(userName, password);
+        console.log(token)
+        return res.status(201).json({ accessToken : token })
```

Comment:
```suggestion
        const accessToken = await userService.getUserById(userName, password);
        
        return res.status(201).json({ accessToken })
```
javascript object에서는 property name과 value의 변수 name이 같다면 위와 같이 간단하게 작성할 수 있고 보다 깔끔한 코드를 작성하는데 도움됩니다 :)